### PR TITLE
Create _child_student_create_account

### PR DESCRIPTION
**Pull request by KatherineCox**
Here's a draft of instructions for students to sign up for an account and send their username to their instructor.  I thought it would be nice to have something that could be dropped in to GDSCN books.  Feedback welcome!  I took bits from several of the existing books.

I went with making the steps H3s, because that seems to be the most common in existing books, usually looking something like:
- H1 Student Activity Guide
    - H2 Account Setup
        - H3 the steps of this child

(I have temporarily set the parent Rmd to point to this branch, so that the child will show up in the preview for this PR (chapter 8); I'll switch it before merging to main)

### CODE REVIEW

File: child/_child_student_create_account.Rmd [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- Alternatively, if you would like to create a Google account that is associated with an existing non-Gmail email address, you can follow [these instructions](https://support.terra.bio/hc/en-us/articles/360029186611).
+
+### Log In to Terra
+
+Next, make sure you can log in to Terra -- you will use Terra to perform computations on AnVIL. 
+
+You can access Terra by going to [`anvil.terra.bio`](https://anvil.terra.bio/), or by clicking the link on the [AnVIL home page](https://anvilproject.org/).
+
+
+```{r, echo=FALSE, fig.alt='Screenshot of the AnVIL home page. The button to "launch Terra" is highlighted.'}
```
Comment (avahoffman) [inline child/_child_student_create_account.Rmd:25]:
```suggestion
```{r, echo=FALSE, fig.alt='Screenshot of the AnVIL home page. The button to "launch Terra" is highlighted.', out.width = '80%', fig.align = 'center'}
```

**Review by avahoffman — APPROVED** [review]
LGTM! Might want to format / center the anvil screenshot, but no other comments!

File: child/_child_student_create_account.Rmd (hunk 1 above)
Comment (ehumph) [reply child/_child_student_create_account.Rmd:25]: I agree, the rest of it looks good!

**Review by ehumph — APPROVED** [review]

File: child/_child_student_create_account.Rmd (hunk 1 above)
Comment (KatherineCox) [reply child/_child_student_create_account.Rmd:25]: Good call, thanks!

**Pull request merged**